### PR TITLE
document persist replace in API section #539

### LINKED ISSUE #539: document persist replace in API section
we need to document persist-replace: https://jupysql.ploomber.io/en/latest/integrations/pandas.html#persist-replace

in the magic-sql.md file: https://jupysql.ploomber.io/en/latest/api/magic-sql.html

### PR DESCRIPTION
**Pull request by anupam-tiwari**
## Describe your changes
Added Presist replace to API section
## Issue number

Closes #539

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)

----
:books: Documentation preview :books:: https://jupysql--602.org.readthedocs.build/en/602/

### CODE REVIEW

**Comment by anupam-tiwari** [comment]
Will sort these issues!

**Comment by edublancas** [comment]
looks like you have a typo `./doc/api/magic-sql.md:211: Presist ==> Persist`

**Comment by edublancas** [comment]
please merge from the main branch to the branch you're working on to fix the issue (this is some edge case we have in our CI setup that we haven't fixed yet)

**Comment by anupam-tiwari** [comment]
I figured the problem, I was not linking #replace-table to the particular section, when you click it scrolls down to that section, I there a way to do that?

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@
 * [Doc] Snowflake integration guide (#384)
 * [Doc] User guide on using JupySQL in `.py` scripts (#449)
 * [Fix] Validating arguments passed to `%%sql` (#561)
+* [Doc] Document persist replace in API section (#539)
```
Comment (neelasha23) [inline CHANGELOG.md:15]: `persist replace` --> `--persist-replace`

File: doc/api/magic-plot.md [hunk 2]
```diff
@@ -152,10 +152,11 @@ generate histograms without explicitly removing NULL entries.
 `%sqlplot` returns a `matplotlib.Axes` object.
 
 ```{code-cell} ipython3
-ax = %sqlplot histogram --table penguins.csv --column body_mass_g
```
Comment (neelasha23) [inline doc/api/magic-plot.md:155]: Was this changed added from `pkgmt format`?

File: doc/api/magic-sql.md [hunk 3]
```diff
@@ -204,6 +208,20 @@ my_data = pd.DataFrame({"x": range(3, 6), "y": range(3, 6)})
 %sql SELECT * FROM my_data
 ```
 
+## Persist replace to table
+
+```{code-cell} ipython3
+my_data = pd.DataFrame({"x": range(3), "y": range(3)})
```
Comment (neelasha23) [inline doc/api/magic-sql.md:214]: my_data -> df

File: doc/howto/ggplot-interact.md [hunk 4]
```diff
@@ -193,8 +193,11 @@ show_legend = widgets.ToggleButton(
 
 ```{code-cell} ipython3
 def plot(b, cmap, show_legend):
-    (ggplot("diamonds", aes(x="price")) + geom_histogram(bins=b, fill="cut", cmap=cmap)
-     + facet_wrap("color", legend=show_legend))
+    (
```
Comment (neelasha23) [inline doc/howto/ggplot-interact.md:196]: Please revert these formatting changes. It's not part of this PR. I see a lot of added newlines as well. 
Please run `pkgmt format` before pushing code

**Comment by neelasha23** [comment]
Please fix docs build and the CI , and request for review again.
@anupam-tiwari

File: doc/howto/ggplot-interact.md (hunk 4 above)
Comment (anupam-tiwari) [reply doc/howto/ggplot-interact.md:196]: I did pkgmt format and pkgmt lint before committing, need to look what is breaking
